src/js/player.js

```diff
@@ -1570,20 +1570,24 @@ class Player extends Component {
   /**
    * start media playback
    *
-   * @return {Player}
-   *         A reference to the player object this function was called on
+   * @return {Promise|undefined}
+   *         Returns a `Promise` if the browser returns one, for most browsers this will
+   *         return undefined.
    */
   play() {
     // Only calls the tech's play if we already have a src loaded
     if (this.src() || this.currentSrc()) {
-      this.techCall_('play');
-    } else {
-      this.tech_.one('loadstart', function() {
-        this.play();
-      });
+      return this.techGet_('play');
```

Comment:
oh, this is an interesting case for the techget/techcall dichotomy. `play` is still technically a setter, which is why `techcall` is used, but we want its return value. I wonder if we need to get rid the the difference. This is particularly important because of how middleware work. We're bubbling `techCall`s from the player to the tech and `techGet`s from the tech to the player.